tests/test_mill_family.py

```
@@ -0,0 +1,548 @@
+            CACHE_STAMPEDE,
+        )
+        k8s_controls = [
+            episode(
+                f"kcl-r0{index}-probe",
+                "CrashLoopBackOff liveness probe container restart",
+                K8S,
+            )
+            for index in (1, 2)
+        ]
+        mills = index_of(
+            (CACHE_STAMPEDE, STAMPEDE_CONTROLS + [stray]),
```

Ruff (0.16.2)

[warning] Consider `[*STAMPEDE_CONTROLS, stray]` instead of concatenation

Replace with `[*STAMPEDE_CONTROLS, stray]`

(RUF005)